pages/common/web-ext.md

```diff
@@ -0,0 +1,36 @@
+# web-ext
+
+> A command line tool for managing web extension development.
+> Homepage: <https://www.npmjs.com/package/web-ext>.
+
+- Open the current web extension in Firefox:
+
+`web-ext run`
+
+- Open a specific directory's web extension in Firefox:
```

Comment:
```suggestion
- Run a web extension from a specific directory in Firefox:
```

Reply:
This and the previous comment are literally what I had when I originally wrote the page. :yum: I'll apply these changes in the morning. :+1: Thanks.